_cards/2021-11-26-0501-Alternate-Accounting-Concepts,-Policies-and-Principles-Entry.md

```diff
@@ -11,13 +11,13 @@ functional_competency_designation: Job Specific Competencies
 competency: Accounting Concepts, Policies and Principles
 competency_group: Alternate
 competency_description: Apply federal accounting standards, fiscal law, policies, regulations, principles, standards, internal controls and procedures to financial management activities
-level: "7-9"
+level: "Entry&#58; (7-9)"
```

Comment:
was this part of the scope ?

Reply:
No, that was a mistake on my part. It seems it's been added to scope though